### PR TITLE
Show Twitter user even if we don't have in db yet

### PR DESCRIPTION
**Pull request by DaleMcGrew**
If someone tries to view a Twitter handle (e.g. https://wevote.me/DaleMcGrew) that isn't in the database, we reach out to Twitter and retrieve and display the information for that account so we can ask the visitor some questions about what to do. 

Also fixed bug with call to twitterIdentityRetrieve.

### CODE REVIEW

File: src/js/routes/Guide/UnknownTwitterAccount.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  static propTypes = {
+    twitter_handle: PropTypes.string,
+    twitter_description: PropTypes.string,
+    twitter_followers_count: PropTypes.number,
+    twitter_photo_url: PropTypes.string,
+    twitter_user_website: PropTypes.string,
+    twitter_name: PropTypes.string
+  };
+
+  constructor (props) {
```
Comment (lisamcho) [inline src/js/routes/Guide/UnknownTwitterAccount.jsx]: This method does nothing. It is already inherited from Component. Should remove.

File: src/js/routes/Guide/UnknownTwitterAccount.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    super(props);
+  }
+
+  render () {
+    let {twitter_handle, twitter_description, twitter_followers_count,
+      twitter_photo_url, twitter_user_website,
+      twitter_name} = this.props;
+
+    // If the displayName is in the twitterDescription, remove it from twitterDescription
+    let displayName = twitter_name ? twitter_name : "";
```
Comment (lisamcho) [inline src/js/routes/Guide/UnknownTwitterAccount.jsx]: A cleaner syntax would be:
``` const displayName = twitter_name || "";
```

File: src/js/routes/Guide/UnknownTwitterAccount.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            {twitter_followers_count ?
+              <span className="twitter-followers__badge">
+                <span className="fa fa-twitter twitter-followers__icon"></span>
+                {numberWithCommas(twitter_followers_count)}
+              </span> :
+              null
+            }
+            &nbsp;&nbsp;
+            <a href={twitter_user_website} target="_blank">Website</a><br />
+            {/*5 of your friends follow Organization Name<br />*/}
```
Comment (lisamcho) [inline src/js/routes/Guide/UnknownTwitterAccount.jsx]: We shouldn't add commented-out code to the repo if features aren't ready.

**Comment by DaleMcGrew** [comment]
I believe I have made all of the changes suggested by @lisamcho.

**Comment by DaleMcGrew** [comment]
Thank you Lisa for the code review. I am merging.

**Pull request merged**